aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputStream.java

```diff
@@ -44,10 +48,27 @@
   private long next = 0;
   private boolean closed = false;
 
+  private final Counter<Long> readBytes;
+  private final Counter<Integer> readOperations;
+
   public OSSInputStream(OSS client, OSSURI uri) {
     this.client = client;
     this.uri = uri;
     this.createStack = Thread.currentThread().getStackTrace();
+
+    this.readBytes = MetricsContext.nullMetrics().counter(
+        FileIOMetricsContext.READ_BYTES, Long.class, Unit.BYTES);
+    this.readOperations = MetricsContext.nullMetrics().counter(
+        FileIOMetricsContext.READ_OPERATIONS, Integer.class, Unit.COUNT);
```

Comment:
Can this constructor call `this(client, uri, MetricsContext.nullMetrics());` instead?

Reply:
Thanks, I have made the changes.